tests/fixtures/manifests/simple.wadm.yaml

```diff
@@ -7,12 +7,27 @@ metadata:
     version: v0.0.1
     description: Sample manifest that passes
 spec:
+  policies:
+    - name: vault-for-component
+      type: secrets.wasmcloud.dev/v1/vault
+      properties:
+        role-name: "test"
+        backend: "vault-prod"
+
   components:
     - name: http-component
       type: component
+      secrets:
+        - backend: "vault-for-component"
+          key: "secret/test"
+
       properties:
         image: ghcr.io/wasmcloud/component-http-hello-world:0.1.0
       traits:
         - type: spreadscaler
           properties:
             instances: 1
+
+            # {"name":"", "type": "wasmcloud.dev/secret/v1", key: "", "policy": "\"{\"role-name\":\"test\"}\"", "backend: ""}
+            # {"name":"", "type": "wasmcloud.dev/secret/v1", key: "", "policy": "whatever", "backend: ""}
+            # POLICY_$app-name-$policy-name
```

Comment:
Looks like maybe some brainstorming code we had 😄

Reply:
Definitely was. Removed!